build.gradle

```diff
@@ -4,11 +4,12 @@ buildscript {
     repositories {
         jcenter()
         mavenLocal()
+        maven { url 'https://maven.fabric.io/public' }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.2'
+        classpath 'com.android.tools.build:gradle:2.1.3'
         classpath 'com.google.gms:google-services:3.0.0'
-
+        classpath 'io.fabric.tools:gradle:1.+'
```

Comment:
Yeah I think they do it to keep their docs always correct, but this will make our builds happier.  Thanks!